tools/driver/swift_symbolgraph_extract_main.cpp

```diff
@@ -260,16 +260,16 @@ int swift_symbolgraph_extract_main(ArrayRef<const char *> Args, const char *Argv
   // don't need to print these errors.
   CI.removeDiagnosticConsumer(&DiagPrinter);
   
-  for (const auto &ModuleName : VisibleModuleNames) {
-    if (ModuleName.str().startswith("_")) {
-      auto CIM = CI.getASTContext().getModuleByName(ModuleName.str());
-      if (CIM && CIM->isCrossImportOverlayOf(M)) {
-        const auto &CIMainFile = CIM->getMainFile(FileUnitKind::SerializedAST);
-        llvm::errs() << "Emitting symbol graph for cross-import overlay module file: "
-          << CIMainFile.getModuleDefiningPath() << '\n';
-        
-        Success |= symbolgraphgen::emitSymbolGraphForModule(CIM, Options);
-      }
+  SmallVector<ModuleDecl *, 2> Overlays;
+  M->findDeclaredCrossImportOverlaysTransitive(Overlays);
+  for (const auto *OM : Overlays) {
```

Comment:
Looks like we used to sort the overlays by name. Is this something we should do here as well? I'm not sure whether sorting was a requirement for us to produce correct SGFs.

Reply:
The original use for `VisibleModuleNames` is for reporting to the user when the given module name wasn't found. The sorting was (as far as i know) just for ease of user experience when reading the module list. I can move the initialization and sorting back into that error branch so we don't hit that in the normal path at all.